docs/REST API/rest-api.md

```diff
@@ -3002,6 +3002,10 @@ Requires [Fleetd](https://fleetdm.com/docs/using-fleet/fleetd), the osquery mana
 {
   "host_id": 1,
   "device_mapping": [
+    {
+      "email": "user@example.com",
+      "source": "idp"
```

Comment:
```suggestion
      "source": "identity_provider"
```

I think we should try to avoid acronyms in the API for clarity. We want Fleet to be easy to use and understand.

FYI I think we should do that same in the UI and use "Identity Provider." I updated [Figma](https://www.figma.com/file/WYkQ6zhSqmSwCVnGHS2XzT/%2313034-Lookup-hosts-based-on-IdP-email?type=design&node-id=101%3A208&mode=design&t=Ic8BCKIOaoHz5Yeh-1):

<img width="220" alt="Screenshot 2023-11-25 at 10 59 50 AM" src="https://github.com/fleetdm/fleet/assets/47070608/0acae42c-366d-4d2a-bdc9-f67be14eec4a">

Reply:
I agree, it is way clearer.